scripts/new-rpm-release

```diff
@@ -18,20 +18,22 @@ set -e
 
 SPEC_FILE="rpm/containerd.spec"
 export SPEC_FILE
-source scripts/.rpm-helpers
+
+# shellcheck disable=SC1090
+source "$(dirname "$0")/.rpm-helpers"
```

Comment:
I guess it's `. "$(dirname "$0")/.rpm-helpers"` instead?

Reply:
Ah, yes, could update it; this one still required Bash though, so there's still other things that are non-POSIX

Reply:
Oh, actually; looks like those were no longer needed; let me update 👍

Reply:
Updated 👍